tests/acceptance/users_frontend.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Users Frontend
+  In order to manage users account in the web
+  As a user
+  I need to check user login and registration in joomla! CMS
+
+  Background:
+    Given I see the joomla! Home page
```

Comment:
Yes, I think background scenario is not necessary here because our users.feature last scenario `User settings (Allow user registration)` display joomla home page.